pom.xml

```diff
@@ -36,15 +37,23 @@ SAX2 and Stax2 APIs
         <url>https://github.com/FasterXML/woodstox</url>
         <tag>HEAD</tag>
     </scm>
-
+	-->
+	
     <properties>
-      <version.msv>2013.6.1</version.msv>
+      <version.msv>2022.8-SNAPSHOT</version.msv>
       <version.plugin.javadoc>3.1.1</version.plugin.javadoc>
 
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>        
+      <skipTests>false</skipTests>
+        
       <!-- Woodstox 5.0 is 1.6+ -->
-      <javac.src.version>1.6</javac.src.version>
-      <javac.target.version>1.6</javac.target.version>
+      <jdk.version>1.8</jdk.version>
```

Comment:
Will then need to go in after #134 is done (which itself is after 6.6.0 release). Probably not a problem as MSV-core change may take a while.